### PR TITLE
Add error for pushing from non-master branch.

### PR DESCRIPTION
**Pull request by shammamah**

### CODE REVIEW

File: tutorial/dash_deployment_server_examples.py [hunk 1]
```diff
@@ -1758,6 +1758,22 @@ def display_instructions2(method):
...
+        dcc.SyntaxHighlighter(s(
+            '''...
+            remote:            Adding BUILD_ENV to build environment...
+            remote:            Unable to select a buildpack'''),
+                              customStyle=styles.code_container, language='python'),
+        dcc.Markdown(s(
+            ''''
+            &nbsp;
+
+            This error might occur if you are trying to push from a branch that is not your `master` branch. Get the name of your current branch by running `git branch`. Then, to push from this branch to the remote server, run `git push plotly your-branch-name:master`.
```
Comment (chriddyp) [inline tutorial/dash_deployment_server_examples.py]: Could we break this into multiple lines?

**Comment by chriddyp** [comment]
Looks like there is a typo, it's causing tests to fail

**Comment by Marc-Andre-Rivet** [comment]
@shammamah Is this PR still alive?

**Comment by shammamah** [comment]
@Marc-Andre-Rivet Working on it right now!

**Comment by shammamah** [comment]
@Marc-Andre-Rivet I'm not sure what the visual changes are, and I don't have access to view the details. How can I fix these issues?

**Comment by Marc-Andre-Rivet** [comment]
@shammamah The difference is what would be expected (the added entry) + some dates changing from the previous snapshot (I've already approved those)

image

Someone should give you access shortly to the Percy snapshots for dash-docs.

Otherwise this looks fine to me. Will just ping @gzork and @chriddyp so they can have another look at the fixed version before merging.

**Pull request merged**